frontend/src/joi_schemas/editGroup.js

```diff
@@ -0,0 +1,13 @@
+import Joi from 'joi';
+
+const schema = Joi.alternatives().try(
+  Joi.object().keys({
+    name: Joi.string().label('Collective name'),
+    logo: Joi.string().uri().allow('').label('Logo url'),
+    mission: Joi.string().max(100).label('Mission'),
+    description: Joi.string().max(255).label('Short description'),
+    longDescription: Joi.string().max(1000).label('Full description'),
```

Comment:
`longDescription` is of type `TEXT`, it doesn't have a limit. In fact, the `longDescription` of BrusselsTogether is 5,596 characters long :-o